react-ui/src/pages/contest/ContestAppBarMain.jsx

```diff
@@ -24,11 +26,9 @@ export default function ContestAppBarMain({ handleVotingExpired, handleReload, c
   } = contest;
 
   if (!name) {
-    return null;
+    return <RouterLinkIconButton state={{ date }} to={backLink} />;
```

Comment:
Hm yeah, I sort of forget what I introduced `!name`, but I think it was because I felt name *had* to be there or we had bigger problems to investigate. I took it out.

Reply:
I thought `!name` was there from when I added the code. It was generally a substitute for "have fetched data" since name was always going to be included, but it's cleaner now that we always include the back button.

Reply:
Ah ok, sounds good thought it might have been from my refactoring